<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProposalDetailDialog</class>
 <widget class="QDialog" name="ProposalDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>543</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>81</y>
      <width>460</width>
      <height>380</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>330</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>提案详情</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="infoBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{font:10px &quot;微软雅黑&quot;;background:transparent;border:none;color: rgb(137,129,161);} 
QToolButton::checked{color:rgb(84,61,137);border-bottom:2px solid rgb(84,61,137);}</string>
     </property>
     <property name="text">
      <string>提案信息</string>
     </property>
    </widget>
    <widget class="QToolButton" name="voteStateBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{font:10px &quot;微软雅黑&quot;;background:transparent;border:none;color: rgb(137,129,161);} 
QToolButton::checked{color:rgb(84,61,137);border-bottom:2px solid rgb(84,61,137);}</string>
     </property>
     <property name="text">
      <string>投票状态</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>460</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="styleSheet">
       <string notr="true">QLabel{font:  12px &quot;微软雅黑&quot;;color: rgb(51, 51, 51);}

</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>提案ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="proposalIdLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>发起人ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="expirationTimeLabel">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>140</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>失效时间</string>
       </property>
      </widget>
      <widget class="QLabel" name="proposerIdLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>提案类型</string>
       </property>
      </widget>
      <widget class="QLabel" name="typeLabel">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>140</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QStackedWidget" name="typeStackedWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>460</width>
         <height>150</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>5</number>
       </property>
       <widget class="QWidget" name="hotColdTransferPage">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>资产类型</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>发送方</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>80</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>接收方</string>
         </property>
        </widget>
        <widget class="QLabel" name="assetLabel_hotCold">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>120</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
        <widget class="QLabel" name="sendLabel_hotCold">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>300</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
        <widget class="QLabel" name="receiveLabel_hotCold">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>80</y>
           <width>300</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
        <widget class="QLabel" name="amountLabel_hotCold">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>20</y>
           <width>120</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>20</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>发送数量</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>备注</string>
         </property>
        </widget>
        <widget class="QLabel" name="memoLabel_hotCold">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>110</y>
           <width>300</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="updateMultisigPage">
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>资产类型</string>
         </property>
        </widget>
        <widget class="QLabel" name="assetLabel_updateMS">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>20</y>
           <width>120</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
        <widget class="QLabel" name="coldAddressLabel_updateMS">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>50</y>
           <width>300</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>冷钱包地址</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>80</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>热钱包地址</string>
         </property>
        </widget>
        <widget class="QLabel" name="hotAddressLabel_updateMS">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>80</y>
           <width>300</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="setPublisherPage">
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>喂价者</string>
         </property>
        </widget>
        <widget class="QLabel" name="assetLabel_publisher">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>20</y>
           <width>120</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
        <widget class="QLabel" name="publisherLabel_publisher">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>50</y>
           <width>300</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>资产类型</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="cancelColdHotPage">
        <widget class="QLabel" name="coldHotTrxIdLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>300</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>200</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>冷热钱包转账交易ID</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="cancelWithdrawPage">
        <widget class="QLabel" name="withdrawTrxIdLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>300</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>200</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>提现交易ID</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="createGuardPage">
        <widget class="QLabel" name="isFormalLabel">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>20</y>
           <width>120</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>是否正式</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Senator地址</string>
         </property>
        </widget>
        <widget class="QLabel" name="guardAddressLabel">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>50</y>
           <width>300</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QTableWidget" name="voteStateTableWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>400</width>
         <height>190</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>19</width>
         <height>12</height>
        </size>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>所需投票者</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>投票状态</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
